cypress/integration/article-view-spec.js

```diff
@@ -177,7 +177,9 @@ describe('Article view', () => {
   it('check text size remains after changing on preview', () => {
     goToCatArticle()
     articlePage.getArticleText().should('have.attr', 'style', 'font-size: 16px;')
-    cy.downArrow().downArrow().downArrow().enter()
+    articlePage.selectOptionFromActionsMenu('sections')
```

Comment:
you found the issue? awesome!
I was confused because it was working fine on my local...

Reply:
I was confused too

Reply:
chrome works, but the user sees the page as blank, but able to select the preview.
but i'm not sure how it looks like in the terminal cypress

Reply:
![image](https://user-images.githubusercontent.com/2560096/83485045-7accf400-a4a6-11ea-8ab5-24044ed1a54a.png)

hmmm when checking the video, it opens the reference preview not article preview ...

Reply:
Ok, I have a slightly changed to the behaviour here, due to the window.screen reason, we don't know where it goes when press down 3 times, therefore press 1 time to get more predictable location either next page or next section.